### DISCUSSION TITLE
Rule on Switchstate does not trigger on Shelly i3

### DISCUSSION THREAD

**Discussion by michifischer** [arendst/Tasmota#19853] [Q&A] [answered]
Hi!
I have a Shelly i3 with tasmota 13.2.0 using this template:
```
{"NAME":"Shelly i3","GPIO":[0,0,0,0,0,320,0,0,193,194,192,0,0,4736],"FLAG":0,"BASE":18}
```

I have
```
switchmode1 1
setoption114 on
rule1 on switch1#state=1 do websend [myhostname.domain] /cm?cmnd=Power1%20TOGGLE done
Rule1 on
```

When I press the pushbutton connected to the I1 cable the console output shows
```
17:55:06.367 RSL: RESULT = {"Switch1":{"Action":"ON"}}
17:55:06.871 RSL: RESULT = {"Switch1":{"Action":"OFF"}}
```
but rule1 does not trigger.

Any ideas what's wrong?

**Comment by sfromis** [comment, marked as answer]
Your rule is invalid, missing `endon`, and that `done` is probably not what you wanted.

Looks like you want to send a command to another device, that can be done simpler like:
`websend [myhostname.domain] Power toggle`
Then Tasmota formats the URL.

**Reply by michifischer** [reply]
omg, I'm so stupid. Too much bash-scripting for me ;-)

Thanks alot, that did the trick!
